scrabble.js

```diff
@@ -1,16 +1,253 @@
-
 const Scrabble = {
+
+  getPoint(letter) {
+    let point = 0;
+    if (['A', 'E', 'I', 'O', 'U', 'L', 'N', 'R', 'S', 'T'].includes(letter)) {
+      point = 1;
```

Comment:
I like that you've encapsulated this in a function. However, given that you're mapping strings to numbers here, is there a data structure that would do this work more simply? Maybe something like this:

```javascript
const LETTER_VALUES = {
  a: 1,
  b: 3,
  // ...
}
```

To put it in terms of computational complexity, in the worst case ('Z') you have to loop through all of these arrays checking for conclusion, which means the algorithm is `O(n)`, where `n` is the number of letters in the alphabet. Looking up a value in a hash, on the other hand, is `O(1)`.